packages/clay-dropdown/src/ClayDropdown.soy

```diff
@@ -19,10 +19,12 @@
 	{@param? searchable: bool}
 	{@param? style: string}
 	{@param? triggerClasses: string}
+	{@param? triggerSize: string}
 
   {let $trigger kind="html"}
     {if $icon}
       {call ClayIcon.render}
+        {param elementClasses: $label ? 'inline-item inline-item-before' : '' /}
```

Comment:
hey @carloslancha, Since we added this with the icon before, would it make sense to add to an icon after the label?  `inline-item inline-item-after`.

Reply:
I'd not add it until somebody needs it

Reply:
Lexicon use cases seem to cover this.
<img width="322" alt="screen shot 2018-02-21 at 11 22 46" src="https://user-images.githubusercontent.com/13750819/36484954-9f97e502-16f9-11e8-8133-8eb27cf3585f.png">

Currently it does not have any spacing, since in ClayButton we have spacing, I think it makes sense to put, to ensure consistency of use of the Buttons.
<img width="187" alt="screen shot 2018-02-21 at 11 24 34" src="https://user-images.githubusercontent.com/13750819/36485016-d131bb92-16f9-11e8-8298-d518c7326191.png">

Reply:
Ok, makes sense

Reply:
But we need to be aware that then everywhere will be a space between label and caret-down, like in management-toolbar, where right now there's no such space

Reply:
Lexicon defined it right, but that doesn't fit with @pat270 markup, so it should have it, just pointing it out.

Reply:
Hmm, Yeah, I think we should look at this more carefully. Hey @pat270 would we cause many problems in adding this?

Reply:
@matuzalemsteles I don't think it would cause major problems as long as spaces are stripped. If you wanted to use a space instead of the margin you could omit `inline-item-after`.

Reply:
Great, thanks @pat270 for the info. 

Hey @carloslancha what do you think?